.travis.yml

```diff
@@ -3,10 +3,11 @@ rbenv:
 - 2.5.3
 addons:
   postgresql: 9.6
+
+env:
+  - RAILS_ENV=test
 script:
-- gem install bundler:2.1.4
-- bundle exec rails db:{create,migrate,seed} RAILS_ENV=test
-- bundle exec rails db:schema:dump RAILS_ENV=test
+- bundle exec rails db:create db:schema:load db:test:prepare
```

Comment:
The magic formuoli! 
How do db:schema:load and db:test:prepare work? What was your process for finding this solution? From some research or from your experience?

Reply:
I got this from different projects I've worked on with my team and I thought it was pretty neat and straightforward.